lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1670,7 +1670,7 @@ def test_certificate_generation_for_students(self):
             'failed': 3,
             'skipped': 2
         }
-        with self.assertNumQueries(175):
+        with self.assertNumQueries(191):
```

Comment:
Agreed - this is still measurably better than it used to be, and will be much better after course grades and "all students in a given course" optimizations are made.